python-api/tests/integrationtests/baseclass.py

```diff
@@ -0,0 +1,412 @@
+"""
+Contains the Base class for integration tests.
+"""
+
+import sys
+import os
+import logging
+import tempfile
+import shlex
+import shutil
+try:
+    import subprocess32 as subprocess
+except ImportError:
+    import subprocess
+
+from glob import glob
+import numpy
+import scipy.misc
```

Comment:
It might be worth putting something like;
```python
try:
  scipy.misc.imread
except AttributeError:
  raise ImportError('Unable to find scipy.misc.imread function, please make sure PIL is installed.')
```